examples/common_vmseries/main.tf

```diff
@@ -198,6 +198,7 @@ module "bootstrap" {
   name                   = each.value.name
   resource_group_name    = try(each.value.resource_group_name, local.resource_group.name)
   location               = var.location
+  storage_acl            = try(each.value.storage_acl, false)
```

Comment:
this property is missing in the bootstrap variable description

this applies to the `main.tf` files in the other (VMSeries based) examples